types/state/DataAccount.go

```diff
@@ -0,0 +1,72 @@
+package state
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/AccumulateNetwork/accumulate/types"
+)
+
+type DataAccount struct {
+	ChainHeader
+	ManagerKeyBookUrl types.String `json:"managerKeyBookUrl"` //this probably should be moved to chain header
```

Comment:
Can we go ahead and put it in the chain header now? We're going to have to wipe the testnet anyways, so we might as well add it to the chain header now.

Reply:
I almost did that, but didn't.  Sure I'll move it over.

Reply:
do we want to maintain the manager keey book as a URL or chainID?  I prefer Url.

Reply:
URL definitely. I think everyone prefers url and Paul seem almost surprised that it wasn't.

Reply:
Almost surprised....  Nothing surprises me (why does that work better with a haughty english accent?)

Yes URLs.  Because chainIDs only work within the context of an ADI (where all components must be on the same BVN now and forever).  If a ChainID isn't defined on a BVN, it becomes a search to find where it is defined.